docs/cli.html

```diff
@@ -1147,7 +1147,8 @@ <h2 id="test">test</h2>
       <blockquote>
 <p>Tests your app via <code>npm test</code>.</p>
 </blockquote><p>  <strong>Usage:</strong> <code>zapier test</code></p><p>  This command is effectively the same as <code>npm test</code>, except we wire in some custom tests to validate your app. We recommend using mocha as your testing framework.</p><p><strong>Arguments</strong></p><ul>
-<li><code>--debug</code> -- <em>optional</em>, print zapier detailed logs to standard out</li>
+<li><code>--quiet</code> -- <em>optional</em>, do not print zapier detailed logs to standard out</li>
```

Comment:
This is strange. It's like the changes from https://github.com/zapier/zapier-platform-cli/pull/71 aren't included in this PR.